Algorithm.Python/SmartInsiderDataAlgorithm.py

```diff
@@ -1,64 +0,0 @@
-# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
-# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from AlgorithmImports import *
-
-### <summary>
-### Example algorithm demonstrating usage of SmartInsider data
-### </summary>
-### <meta name="tag" content="using data" />
-### <meta name="tag" content="custom data" />
-### <meta name="tag" content="smart insider" />
-### <meta name="tag" content="form 4" />
-### <meta name="tag" content="insider trading" />
-class SmartInsiderDataAlgoritm(QCAlgorithm):
-    def Initialize(self):
-        '''Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.'''
-
-        self.SetStartDate(2019, 7, 25)
-        self.SetEndDate(2019, 8, 2)
-        self.SetCash(100000)
-
-        self.symbol = self.AddEquity("KO", Resolution.Daily).Symbol
-        self.AddData(SmartInsiderTransaction, "KO")
-        self.AddData(SmartInsiderIntention, "KO")
```

Comment:
Repo at https://github.com/QuantConnect/Lean.DataSource.SmartInsiderIntentionsTransactions only has `Demonstration.cs` think we should keep both `.py` and `.cs` example algorithms

Reply:
Same for all repos 💯